File: yurt-system/yurt-system-server/src/main/java/com/github/liulus/yurt/system/repository/ParamRepository.java
package com.github.liulus.yurt.system.repository;

import com.github.liulus.yurt.system.model.entity.SysParam;
import com.github.liulus.yurt.jdbc.JdbcRepository;

/**
 * @author liulu
 * @version 1.0
 * created_at 2020/5/26
 */
public interface ParamRepository extends JdbcRepository<SysParam> {
    SysParam selectByCode(String code);
}

File: yurt-system/yurt-system-server/src/main/kotlin/com/github/liulus/yurt/system/repository/ParamRepository.kt
package com.github.liulus.yurt.system.repository

import com.github.liulus.yurt.jdbc.JdbcRepository
import com.github.liulus.yurt.system.model.entity.SysParam

/**
 * @author liulu
 * @version 1.0
 * created_at 2020/5/26
 */
interface ParamRepository : JdbcRepository<SysParam?> {
    fun selectByCode(code: String?): SysParam?
}